data/ui/headerbar.blp

```diff
@@ -85,6 +85,11 @@ menu primary_menu {
       action: 'app.hide_offline';
       label: _("_Hide unavailable books");
     }
+
+    item {
+      action: 'app.hide_read';
+      label: _("_Hide read books");
```

Comment:
`H` as a mnemonic clashes with the other action's `H` from above. I'd consider using the `r` here.